rio_tiler/io/rasterio.py

```diff
@@ -271,7 +271,7 @@ def statistics(
         self,
         categorical: bool = False,
         categories: Optional[List[float]] = None,
-        percentiles: List[int] = [2, 98],
+        percentiles: Optional[List[int]] = None,
```

Comment:
We defaults to the values set in the `get_array_statistics` function. Setting a default here is unnecessary